lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -46,19 +46,34 @@ class CategoryRenderer {
   get _clumpDisplayInfo() {
     return {
       failed: {
+        title: '',
         className: 'lh-clump--failed',
+        expandable: false,
+        expandByDefault: false,
```

Comment:
arg, sorry, I was hoping you'd wait for #6930. I think a lot of this file can be deleted. I'm not sure it even makes sense to have clumps anymore since groups and clumps won't be at all alike anymore.

Reply:
ok, i'll wait for that. third times the charm